Neos.Media/Classes/Domain/Model/AssetSource/AssetProxy/SupportsIptcMetadata.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Neos\Media\Domain\Model\AssetSource\AssetProxy;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+  *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Interface for an Asset Proxy which supports IPTC Metadata
+ *
+ * See https://iptc.org/standards/iim/
+ *
+ * Note that the property names are UpperCamelCase and as defined in the IPTC IIM specification.
+ * Examples: "Title", "Keywords", "CopyrightNotice"
+ */
+interface SupportsIptcMetadata
```

Comment:
See above

Reply:
Shouldn't there be a separate `IptcMetadataInterface` and the respective Asset just have a `getIptcMetadata`?

Reply:
nah, I'd rather use adjectives still, where possible.

Reply:
No, what I meant is, shouldn't metadata be a separate object, so the asset can only give you the metadata object and those methods are actually in that separate object?